shoop/admin/modules/taxes/views/edit.py

```diff
@@ -4,16 +4,24 @@
 #
 # This source code is licensed under the AGPLv3 license found in the
 # LICENSE file in the root directory of this source tree.
+from django.utils.translation import ugettext_lazy as _
+
+from shoop.admin.base import MenuEntry
 from shoop.admin.forms.fields import PercentageField
 from shoop.admin.utils.views import CreateOrUpdateView
 from shoop.core.models import CustomerTaxGroup, Tax, TaxClass
 from shoop.utils.multilanguage_model_form import MultiLanguageModelForm
 
 
+class _Breadcrumbed(object):
```

Comment:
I'm not very happy about Yet Another Mixin Class, but I think we can get rid of this (for instance by making `Breadcrumbs.infer()` smarter) later.